deploy/kne/kind.yaml

```diff
@@ -0,0 +1,18 @@
+cluster:
+  kind: Kind
+  spec: 
+    name: kne
+    recycle: True
+    version: 0.11.1
+    image: kindest/node:v1.22.0
```

Comment:
I think this needs a "wait" field of 1m. During my testing, the kind cluster creation was much more reliable if it waited (usually less than 20 seconds) to finish setup. Then you would need to override the UnmarshalYAML func

type KindSpec struct {
        Name    string        `yaml:"name"`
        Version string        `yaml:"version"`
        Image   string        `yaml:"image"`
        Retain  bool          `yaml:"retain"`
        Wait    time.Duration `yaml:"wait"`
        Kubecfg string        `yaml:"kubecfg"`
}

func (k *KindSpec) UnmarshalYAML(unmarshal func(interface{}) error) error {
        type alias struct {
                Name    string `yaml:"name"`
                Version string `yaml:"version"`
                Image   string `yaml:"image"`
                Retain  bool   `yaml:"retain"`
                Wait    string `yaml:"wait"`
                Kubecfg string `yaml:"kubecfg"`
        }

        var a alias
        if err := unmarshal(&a); err != nil {
                return err
        }
        t, err := time.ParseDuration(a.Wait)
        if err != nil {
                return err
        }
        k.Name = a.Name
        k.Version = a.Version
        k.Image = a.Image
        k.Retain = a.Retain
        k.Wait = t
        k.Kubecfg = a.Kubecfg
        return nil
}

Reply:
you shouldn't wait you should watch

right now i am just using the fact i get a client as the validation but we can decide what watches should be there like we could check the control plane pod status but so far i don't have an example of it not working